nab/detectors/htmcore/htmcore_detector.py

```diff
@@ -147,6 +148,7 @@ def __init__(self, *args, **kwargs):
     if PANDA_VIS_ENABLED:
         pandaServer.Start()
         self.BuildPandaSystem(parameters_numenta_comparable)
+        self.firstStep = True
```

Comment:
nit, could be, but you could use something like TM (or connections) `.getIterationNum()`

Reply:
i couldn't find iteration getter for TM, but for SP there is. Should i use that?

Reply:
I'd say best would be `connections.iterations()` ..let me see if it's exposed to python- yes.